README.md

```diff
@@ -220,6 +220,33 @@ challenge(bytes32 _sessionId, bytes _proofBlob, bytes _unsignedCommitTx)
 - **bytes _unsignedCommitTx**: // TODO
 
 
+-----------
+# Tests
+
+Install Solc
+
+Mac OS X:
```

Comment:
Isn't this a verbatim copy of what's on solc install guide? Can we not just reference the solidity read the docs, unless there's something special / different about the setup for mac os from e.g. linux

Reply:
otherwise, lgtm

Reply:
@relyt29 Done!